src/acceleration/config/AccelerationConfiguration.cpp

```diff
@@ -492,6 +498,12 @@ void AccelerationConfiguration::addTypeSpecificSubtags(
                                                    VALUE_RESIDUAL_SUM_PRECONDITIONER})
                                       .setDocumentation("The type of the preconditioner.");
     tagPreconditioner.addAttribute(attrPreconditionerType);
+    auto attrpreconditionerUpdateOnThreshold = XMLAttribute<bool>(ATTR_PRECOND_UPDATE_ON_THRESHOLD, true)
+                                                   .setDocumentation("To update the preconditioner weights after the first time window:"
+                                                                     "- `true`: The preconditioner weights are only updated if the relative difference of the weights is larger than the limit.\n"
+                                                                     "- `false`: The preconditioner weights are updated after every iteration.\n"
+                                                                     "When this tag is not provided, the value is set to `true` by default.");
```

Comment:
No need to document the default. This is automatically rendered, see https://precice.org/configuration-xml-reference.html